packages/api/methods.ts

```diff
@@ -192,3 +192,27 @@ export function updateRule(rule) {
 export function deleteRule(id) {
   return send('api/rule-delete', { id });
 }
+
+export function getSchedules() {
+  return send('api/schedules-get');
+}
+
+export function createSchedule(schedule, conditions) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** would you mind adding the types for all these input args (for this method and the others too)?